<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Query was successful. What do you want to do with the result?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Result</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowResult">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>40</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Show result in QGIS layer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveAndShowResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Saver and show result in QGIS layer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
